tests/valgrind.suppression

```diff
@@ -207,6 +207,23 @@
    fun:dlopen@@GLIBC_2.2.5
    fun:elektraModulesLoad
 }
+{
+   <insert_a_suppression_name_here>
+   Memcheck:Leak
+   match-leak-kinds: reachable
+   fun:calloc
+   fun:_dl_new_object
+   fun:_dl_map_object_from_fd
+   fun:_dl_map_object
+   fun:dl_open_worker
+   fun:_dl_catch_exception
+   fun:_dl_open
+   fun:dlopen_doit
+   fun:_dl_catch_exception
+   fun:_dl_catch_error
+   fun:_dlerror_run
+   fun:dlopen@@GLIBC_2.2.5
+}
```

Comment:
Is this suppression too general? I took the output directly from valgrind. My concern is that the suppression is not limited to the gpg2 binary, but I don't know if this can be specified such that it matches only for that binary.

Reply:
Looks very specific to me. You did not include any catch-all, so it should not be possible that such a leak is actually triggered by our code?